ledger/accountdb.go

```diff
@@ -167,6 +171,14 @@ type persistedAccountData struct {
 	round basics.Round
 }
 
+type persistedOnlineAccountData struct {
+	addr              basics.Address
+	accountData       baseOnlineAccountData
+	rowid             int64
```

Comment:
what happens if rowid references an out-of-date address that has been deleted in the last 320 rounds (and then possibly reclaimed by a different address)?